gateway-js/src/__tests__/gateway/reporting.test.ts

```diff
@@ -13,15 +13,6 @@ import { Plugin, Config, Refs } from 'pretty-format';
 import { Report, Trace } from 'apollo-reporting-protobuf';
 import { fixtures } from 'apollo-federation-integration-testsuite';
 
-// TODO: We should fix this another way, but for now adding this
```

Comment:
I don't really understand what's going on with the code here, but this is just a test, so doing stuff to make it pass seems reasonable.